c2corg_ui/templates/article/helpers/view.html

```diff
@@ -31,10 +31,48 @@
       <p><span class="value-title" translate>article_type</span>: <span class="value" x-translate>${article['article_type']}</span></p>
     % endif
 
+    % if article.get('author') and article['article_type'] != 'collab' and article['author']['user_id'] != article['associations']['users'][0]['document_id']:
+      <%
+          user_id = article['author']['user_id']
+          user_lang = article['locales'][0]['lang']
+      %>
+      <p><span class="value-title" translate>author</span>:
+        <a class="value" href="${request.route_path('profiles_view', id=user_id, lang=user_lang)}" >${article['author']['name']}</a>
+    % endif
+
     % if article.get('quality'):
       <p><span class="value-title" translate>quality</span>: <span class="value" x-translate>${article['quality']}</span></p>
     % endif
 
     </span>
 </div>
 </%def>
+
+<%def name="get_article_associated_users(article)">\
```

Comment:
Currently, there is no function to do it but it is allowed in v6_common/associations and old data contain associated users also.